ros2_control_test_assets/CMakeLists.txt

```diff
@@ -26,4 +26,32 @@ install(TARGETS ros2_control_test_assets
 )
 
 ament_export_targets(export_ros2_control_test_assets HAS_LIBRARY_TARGET)
+find_package(ament_cmake_gtest REQUIRED)
+find_package(ament_cmake_gmock REQUIRED)
+find_package(ament_cmake REQUIRED)
+find_package(ament_lint_auto REQUIRED)
+find_package(ament_lint_common REQUIRED)
+find_package(ament_index_cpp REQUIRED)
+find_package(rclcpp REQUIRED)
+find_package(ament_cmake_gtest REQUIRED)
+find_package(ament_cmake_gmock REQUIRED)
+find_package(ament_index_cpp REQUIRED)
```

Comment:
duplicates, and we don't need all of them (rclcpp?) here?

Reply:
Sorry, I didn't see them, will delete the unused packages and move to the right place.